backend/collaboration/src/websocket.ts

```diff
@@ -0,0 +1,53 @@
+import 'dotenv/config';
+import { createServer } from 'http';
+import { WebSocketServer } from 'ws';
+import * as Y from 'yjs';
+import { setPersistence, setupWSConnection } from '@/y-postgresql-util/utils';
+import { IWSSharedDoc } from '@/y-postgresql-util/interfaces';
+import { PostgresqlPersistence } from 'y-postgresql';
+import app from './server';
+
+const server = createServer(app);
```

Comment:
Make this whole file as a function then return the server, the main server in `index.ts` should still be imported from `server.ts`.

Split out the PostgresqlPersistence part into its own file, and set the env vars in the main `config.ts` file and import from there.